.azure/scripts/setup-kubernetes.sh

```diff
@@ -80,6 +81,18 @@ if [ "$TEST_CLUSTER" = "minikube" ]; then
     fi
 
     minikube addons enable default-storageclass
+
+    # Add Docker hub credentials to Minikube
+
+    set +ex
```

Comment:
Should the `set +ex` / `set -ex` be inside the `if`?

Reply:
Yeah, right